### PR TITLE
feat: Snackbar added

### LINKED ISSUE #3: [UI] Show a Snackbar on task creation or failure
#### Current Behavior
There is no _snackbar_  is shown after a task gets created or fails to be created.

#### Desired Behavior
Show _snackbar_ after the task gets created or if there is any error.

### PR DESCRIPTION
**Pull request by abhishekpaturkar**
**Description**

This PR fixes #3 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: client/src/components/TaskModal/index.tsx [hunk 1]
```diff
@@ -1,330 +1,336 @@
-"use client";
+"use client"
```
Comment (ritiksaxena124) [inline client/src/components/TaskModal/index.tsx]: Hey @abhishekpaturkar will you revert the unnecessary changes in this file please.

**Review by ritiksaxena124 — COMMENTED** [review]
Revert all unwanted changes please and thanks for taking this issue @abhishekpaturkar, you did great 😎

**Comment by abhishekpaturkar** [comment]
Done

**Comment by ritiksaxena124** [comment]
Thanks @abhishekpaturkar, you did a great job. Time to merge this 🙌🚀
